README.md

```diff
@@ -36,7 +36,12 @@ From Terminal:
 From terminal:
 - Navigate to app's directory: `cd dwellingly-app`
 - Install: `npm install`
+- Build: `npm run build`
```

Comment:
So `npm run build` actually bundles the app for Production mode, in order to be shipped, so I wouldn't include it here

Reply:
Gotcha! I went ahead and removed it and also took out of the alias command in that doc. I had a different understanding of build, so there's a couple questions about it I'd like to ask. I'll message you~